content/tr/cloud-computing.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Bulut Bilişim
+status: Completed
+category: Kavram
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## Nedir
+
+Bulut bilişim, internet üzerinden isteğe bağlı olarak CPU, ağ ve disk kapasiteleri gibi bilişim kaynaklarının sunulduğu, kullanıcıların uzaktaki fiziksel bir konumda bilgi işlem gücüne erişebildiği ve kullanabildiği bir hizmettir. Genellikle, bulut altyapısının bir organizasyona ayrılmış olup olmadığına veya genele açık hizmetlerde paylaşılıp paylaşılmadığına bağlı olarak, özel veya genel bulut şeklinde bir ayrım yapılır.
```

Comment:
```suggestion
Bulut bilişim, internet üzerinden isteğe bağlı olarak CPU, ağ ve disk kapasiteleri gibi bilişim kaynaklarının sunulduğu, kullanıcıların uzaktaki fiziksel bir konumda bilgi işlem gücüne erişebildiği ve kullanabildiği bir hizmettir.
Genellikle, bulut altyapısının bir organizasyona ayrılmış olup olmadığına veya genele açık hizmetlerde paylaşılıp paylaşılmadığına bağlı olarak, özel veya genel bulut şeklinde bir ayrım yapılır.
```

In general, we should target having a line per sentence. See https://glossary.cncf.io/contribute/#best-practices

Reply:
It helps with reviewing the content. Otherwise the lines are too big for IDEs.

Another case... I compare English content with Turkish. If the lines in Turkish content are too long, I can't do a proper comparison.

Reply:
Thanks for the suggestion @aliok!